debian/copyright

```diff
@@ -0,0 +1,17 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: key-mapper
+Upstream-Contact: Sezanzeb <proxima@sezanzeb.de>
+Source: https://github.com/sezanzeb/key-mapper
+
+Files: *
+Copyright: 2020-2021 sezanzeb
+License: GPL-3+
+
+Files: debian/*
+Copyright: 2020-2021 sezanzeb
+  2021 Matthias Blümel <debian@blaimi.de>
+License: GPL-3+
+
+License: GPL-3+
+ On Debian systems the full text of the GNU GPL v2 can be found
+ in the `/usr/share/common-licenses/GPL-3' file.
```

Comment:
You still need the license grant here:

```suggestion
License: GPL-3+
 This package is free software; you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation; either version 3 of the License, or
 (at your option) any later version.
 .
 This package is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
 .
 You should have received a copy of the GNU General Public License
 along with this program. If not, see <http://www.gnu.org/licenses/>
 .
 On Debian systems, the complete text of the GNU General
 Public License version 3 can be found in "/usr/share/common-licenses/GPL-3".
```